packages/core/src/mixins/__tests__/closable.test.js

```diff
@@ -85,7 +85,7 @@ it('intercepts event propagation if instructed', () => {
             <ClosableFoo closable={{ stopEventPropagation: true }} />
         </div>
     );
-    wrapper.find('div#foo').simulate('click');
+    wrapper.find('div.gyp-closable').simulate('click');
```

Comment:
This is because wrapped component (`#foo`) is not the child of `div.gyp-closable` now. To test `stopPropagtion` we have no choice but testing on `div.closable` itself.